packages/custom/products/public/views/cart.html

```diff
@@ -54,7 +54,7 @@
 </div>
  -->
 
-<div class="admin-container">
+<!-- <div class="admin-container">
```

Comment:
you can remove this file I believe Its not needed anymore as we have the file in Carts package